packages/loader/container-loader/src/loader.ts

```diff
@@ -68,30 +72,24 @@ export class RelativeLoader extends EventEmitter implements ILoader {
     public get IFluidRouter(): IFluidRouter { return this; }
 
     public async resolve(request: IRequest): Promise<IContainer> {
-        if (request.url.startsWith("/")) {
-            // If no headers are set that require a reload make use of the same object
-            const container = await this.containerDeferred.promise;
-            return container;
-        }
-
-        return this.loader.resolve(request);
+        assert(request.url.startsWith("/"));
+        // If no headers are set that require a reload make use of the same object
+        return this.containerDeferred.promise;
     }
 
     public async request(request: IRequest): Promise<IResponse> {
+        assert(request.url.startsWith("/"));
```

Comment:
these changes seems breaking, and i'm also worried that the loader on the scope doesn't use the relative loader

Reply:
I'll change it to return the 404 response like Matt suggested?  In the old state where Boehmia was using the relative loader, all the requests pass through to the real loader anyway because they just weren't trying to resolve to the current container...  Imo this kind of goes back to a different issue I had with the loader object, where it's provided in a lot of places as the interface but the consumers *really* care about its implementation.  We could maybe provide an explicit RelativeLoader instead of an ILoader on the data store alongside the optional scope loader, but I'm not sure how useful that actually ends up being